app/services/monthly_district_data_service.rb

```diff
@@ -65,62 +65,66 @@ def header_row
   end
 
   def district_row
-    complete_registration_counts = repo.complete_registration_counts.find { |k, _| k.slug == region.slug }.last
```

Comment:
The changes to this file are in response to a suggestion by @ssrihari to use hashes instead of arrays to make it easier to keep track of the csv contents.